types/node/crypto.d.ts

```diff
@@ -4457,6 +4457,12 @@ declare module "crypto" {
             ): Promise<ArrayBuffer>;
         }
     }
+
+    global {
+        namespace NodeJS {
+            const crypto: webcrypto.Crypto;
```

Comment:
Why did you add `crypto` to the `NodeJS` namespace?

Reply:
To be honest, I simply adhered to what appeared to be a convention, placing global Node-related elements within this namespace

Reply:
Only interfaces or types can be in `NodeJS` namespace since there is no `NodeJS` variable in global context:
```
> console.log(global.NodeJS)
undefined
```